dietpi/dietpi-software

```diff
@@ -1571,6 +1571,12 @@ Available commands:
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/distributed_projects/#ipfs-node'
 		# - RISC-V: Missing package
 		aSOFTWARE_AVAIL_G_HW_ARCH[$software_id,11]=0
+		#-----------------
+		software_id=206
+		aSOFTWARE_NAME[$software_id]='microblog.pub'
+		aSOFTWARE_DESC[$software_id]='A self-hosted, single-user, ActivityPub powered microblog.'
+		aSOFTWARE_CATX[$software_id]=19
```

Comment:
This fits much better to category 6 (Social & Search) where WordPress and phpBB are located.
```suggestion
		aSOFTWARE_CATX[$software_id]=6
```
Please also move this code blog up there.